conans/client/conf/__init__.py

```diff
@@ -34,11 +34,11 @@
     Android:
         api_level: ANY
     iOS:
-        version: ["7.0", "7.1", "8.0", "8.1", "8.2", "8.3", "9.0", "9.1", "9.2", "9.3", "10.0", "10.1", "10.2", "10.3", "11.0"]
+        version: ["7.0", "7.1", "8.0", "8.1", "8.2", "8.3", "9.0", "9.1", "9.2", "9.3", "10.0", "10.1", "10.2", "10.3", "11.0", "11.1", "11.2", "11.3", "11.4", "12.0", "12.1"]
     watchOS:
-        version: ["4.0"]
+        version: ["4.0", "4.0.1", "4.1", "4.2", "4.2.2", "4.2.3", "4.3", "4.3.1", "4.3.2", "5.0", "5.0.1", "5.1", "5.1.1", "5.1.2", "5.1.3"]
```

Comment:
@SSE4 @laundry-96 any insight about the need of managing the "patch" version? for `watchOS` and `tvOS`? our criteria here should be the "compatibility".